prelude/bootstrap.js

```diff
@@ -1241,6 +1243,40 @@ function payloadFileSync(pointer) {
     accessFromSnapshot(path, callback);
   };
 
+  // ///////////////////////////////////////////////////////////////
+  // mkdir /////////////////////////////////////////////////////////
+  // ///////////////////////////////////////////////////////////////
+
+  function mkdirInSnapshot(path_, cb) {
+    var cb2 = cb || rethrow;
+    return cb2(
+      new Error('Cannot mkdir in a snapshot. Try mountpoints instead.')
```

Comment:
Is there a better error for here?

Reply:
I would instead change the name of the method in something more user friendly, something like `mkdirThrowInSnapshot`

Reply:
I went with `mkdirFailInSnapshot` because it doesn't throw if a callback is provided to the async method, but it still fails.